src/abstractops.cpp

```diff
@@ -438,8 +438,12 @@ void AbsFpEncoding::addConstants(const set<llvm::APFloat>& const_set) {
         auto sv_prec_bits = Expr::mkFreshVar(Sort::bvSort(sv_prec_bitwidth),
                             "fp_const_sval_prec_bits_");
 
-        limit_bits = mkNonzero(
-            Expr::mkFreshVar(limit_bits, "fp_const_limit_bits_"));
+        auto inf_sv_value = (1 << sv_prec_bitwidth) -
+                            (1 << value_bit_info.prec_bitwidth);
+        auto limit_var = Expr::mkFreshVar(limit_bits, "fp_const_limit_bits_");
+        limit_bits = Expr::mkIte(sv_prec_bits.uge(inf_sv_value), 
+          limit_var,
```

Comment:
Why is this var not nonZeroed?

Reply:
Consider this case: let's assume that f32_max + epsilon has been assigned the BV 0[SB] 01[LB] 00000[TB] 00[PB]. Then, BVs within and including the range (0 00 11110 00 .. 0 00 11111 11) may represent invalid values that does not exist in real-world double. Therefore we should assign the constants starting from such BVs.